stainless_extraction/src/lib.rs

```diff
@@ -207,7 +207,7 @@ impl<'l, 'tcx> BaseExtractor<'l, 'tcx> {
     def_id: DefId,
     symbol_path: Vec<String>,
   ) -> StainlessSymId<'l> {
-    let path: Vec<String> = symbol_path.iter().map(sanitize_path_name).collect();
+    let path: Vec<String> = symbol_path.iter().map(|n| sanitize_path_name(n)).collect();
```

Comment:
Out of curiosity: why?

Reply:
Clippy suggest changing the method signature to take a `&str` instead of a `String` since ownership is not needed, and we thus have to explicitly pass the argument here because auto-deref does not kick in when passing a function identifier.